src/PersistentVector.jl

```diff
@@ -15,6 +15,7 @@ function PersistentVector{T}(arr) where T
         append(PersistentVector{T}(DenseLeaf{Vector{T}}(), T[], 0), arr)
     end
 end
+PersistentVector{T}(v::PersistentVector{T}) where {T} = v
```

Comment:
I think the convention is this constructor copies!

`convert(PersistentVector, v)` does not if v is a pvec. Could you add that instead?

Reply:
Sure I can do that. Question though - if the data structure is immutable why would we ever want to copy it?

Reply:
Good question, someone may be using `===` or something, but unlikely.

Reply:
But maybe I feel like WebIO should use `convert`.

Reply:
I don't really feel **super** strongly either way but my preference would be to implement the constructor (so leave what we have) and a convert method. And then use the convert method for WebIO. Is that okay?

Or would you prefer to just have the convert method?

Reply:
Either should probably work.